### PR TITLE
Feature updated README and icon

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,277 @@
...
 [![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![License](https://img.shields.io/cocoapods/l/Sugar.svg?style=flat)](http://cocoadocs.org/docsets/Sugar)
 [![Platform](https://img.shields.io/cocoapods/p/Sugar.svg?style=flat)](http://cocoadocs.org/docsets/Sugar)
 
-## Usage
+## Table of Contents
+
+* [iOS](#ios)
+  * [Application](#application)
+  * [iOS Extensions](#ios-extensions)
```
Comment (vadymmarkov) [inline README.md]: Could be just `Extensions` because it's already under `iOS`.

Comment (zenangst) [inline README.md]: How would you solve the #anchor if both have the same name?

Comment (vadymmarkov) [inline README.md]: Oh right. Then it's fine :wink:

**Comment by vadymmarkov** [comment]
@zenangst It's been a readme day for you :smile: You've earned a :cake:

**Pull request merged**